### ISSUE TITLE
choosing a protected note's folder tag won't open the associated note

### ISSUE THREAD

**Issue by chaserene** [sn-extensions/folders-component#53]
for non-protected notes, when you click on a folder tag in the folder sidebar, the first note that has that tag will be selected and opened. for protected ones, nothing happens, you need to click on the note itself to trigger the password dialog. this is inconvenient, even more so when the note list is hidden and you need to toggle it just to open a protected note.

my suggestion would be to trigger the password dialog with the selection of the folder tag.

**Comment by JaspalSuri** [comment]
Hi @chaserene, thanks for the feedback. I can see how it would be inconvenient when the notes list is hidden; I'll discuss this with the dev team to see if it can be changed. Part of the reason why it currently behaves this way is because a user might have pinned a protected note and may not want to deal with the authentication prompt each time that the app has loaded.

**Comment by chaserene** [comment]
@JaspalSuri thanks for discussing this! also thanks for explaining the design rationale. my thinking is that if you pin a note, that means you want to use if often, make it your "home page" in SN. if the top pinned note is a protected note, chances are the user wants to see that note at startup, so they want to unlock it.

**Comment by chaserene** [comment]
another thing is that users who have a protected note as the top pinned note are probably few. meanwhile, using the Folders component with the note list hidden and trying to use note protection is necessarily a more frequent use case.

**Comment by JaspalSuri** [comment]
If we receive more feedback on this issue from other users we'll consider changing its behavior. 🙂

> my thinking is that if you pin a note, that means you want to use if often, make it your "home page" in SN. if the top pinned note is a protected note, chances are the user wants to see that note at startup, so they want to unlock it.

Personally this isn't true for me as I have several pinned notes but want to create new notes most of the time. I have my notes sorted by _Date Modified_, so the note that serves as a clipboard sometimes appears at the top but I don't want to access it every time that I open the app nor necessarily have something to add to the other pinned notes. That said, it's difficult to tell how most of our users use the app without receiving enough feedback. We're thankful that you created this thread, so that other users with similar feedback will be more likely to chime in on this.

**Comment by chaserene** [comment]
that's right. I was thinking too much within the frames of how I use SN :)
